src/ChannelServer/Util/GmCommands.Handlers.cs

```diff
@@ -687,23 +742,7 @@ private CommandResult HandleRecallAll(ChannelConnection conn, Character sender,
 				return CommandResult.Okay;
 			}
 
-			// Recall each player to target's location.
-			var location = target.GetLocation();
-			foreach (var character in characters)
-			{
-				character.Warp(location);
-				character.ServerMessage("You've been warped to {0}'s location.", target.TeamName);
-			}
-
-			if (sender == target)
-			{
-				sender.ServerMessage("You have called {0} characters to your location.", characters.Length);
-			}
-			else
-			{
-				sender.ServerMessage("You have called {0} characters to target's location.", characters.Length);
-				target.ServerMessage("{1} called {0} characters to your location.", characters.Length, sender.TeamName);
-			}
+			RecallCharacters(sender, target, characters);
```

Comment:
This should either have a `this.`, or the method should be static, as per conventions. And wouldn't the function be better placed in the "functions part" of the commands class, instead of in between the handlers?

Reply:
I was thinking about the "functions part" of the commands class, but as you said, it would be better to have handlers and the constructor on the same file, so the function should be there as well.

most likely direclty above or below the functions that use it so it can be found quickly.

I'll add a "this." to it.

Reply:
My line of thought was that the handlers part has the handlers, incl the list of handlers, while the other part has the managing and potential helper functions. Maybe the partial was a mistake, that only complicates placement^^

Reply:
The partial on this place was definitely no mistake. Like I said, I was also thinking about placing the function on the "functions part", and so be it.

Reply:
I mean I'm not sure either^^ Technically you could even put such helper functions into a third partial. I liked that idea, but not it feels complicated =P

Reply:
I wouldn't recommend more than 2 partials, if it is avoidable. Exceptions would be formulars, but we don't use them on melia for now. Let's just let the function where it is right now. The usability is good there where it is.